### PR TITLE
Removing non-json text because it creates invalid json output

### PR DESCRIPTION
**Pull request by riffraff169**
For ticket #859

### CODE REVIEW

File: netctl/netctl.go [hunk 1]
```diff
@@ -65,8 +65,6 @@ func inspectPolicy(ctx *cli.Context) {
 	tenant := ctx.String("tenant")
 	policy := ctx.Args()[0]
 
-	fmt.Printf("Inspeting policy: %s tenant: %s\n", policy, tenant)
```
Comment (dseevr) [inline netctl/netctl.go]: I would consider changing these to `log.Debugf` so they're available if you `DEBUG=1` but won't show up in normal usage

**Comment by dseevr** [comment]
If you change the description to

```
Fixes #859
```

the linked issue will be auto-closed when the PR is merged

File: netctl/netctl.go [hunk 2]
```diff
@@ -844,8 +833,6 @@ func inspectBgp(ctx *cli.Context) {
 
 	hostname := ctx.Args()[0]
 
-	fmt.Printf("netctl. Inspecting bgp: %s\n", hostname)
```
Comment (gaurav-dalvi) [inline netctl/netctl.go]: How does exactly removing these printfs making o/p valid json

Comment (dseevr) [inline netctl/netctl.go]:
```
blah blah
{"foo":"bar"}
```

isn't JSON output that be parsed without manually tidying it up first.  If you request JSON with `-j`, the output should be parseable as JSON

**Pull request closed without merging**